ctapipe/tools/process.py

```diff
@@ -70,6 +76,18 @@ class ProcessorTool(Tool):
             "show a progress bar during event processing",
             "don't show a progress bar during event processing",
         ),
+        **flag(
+            "recompute_dl1",
```

Comment:
I suggest calling this  alias `--recompute-dl1` (using a dash not an underscore) as we do with other command-line arguments.  Same for `--recompute-dl2`